frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -96,7 +96,7 @@ function FileManagerMenu:setUpdateItemTable()
             G_reader_settings:flush()
         end
     }
-    if Device.isKobo() then
+    if Device.isKobo() or (Device.isKindle() and not Screensaver:isKindleSpecialOffers()) then
```

Comment:
For better readability and efficiency, I suggest we create a device method specifically for this purpose. So in `frontend/device/generic/device.lua`, we can define a method called something like `supportsScreensaver()` which returns `false` by default. Then override this method in `frontend/device/kobo/device.lua` and `frontend/device/kindle/device.lua`.